bokeh/server/server.py

```diff
@@ -98,8 +98,8 @@ def get_session(self, app_path, session_id):
     def show(self, app_path, browser=None, new='tab'):
         ''' Opens an app in a browser window or tab.
 
-            Needless to say you could call this on your local desktop but
-            should not call it when running the server on an actual server.
+            Useful for testing server applications on your local desktop but
+            should not call when running bokeh-server on an actual server.
```

Comment:
we may consider mentioning also that you can avoid using it at all if you use bokeh cmd `--show` argument :)